extra_settings/models.py

```diff
@@ -208,7 +216,17 @@ def __init__(self, *args, **kwargs):
             self.value = value
         self.name_initial = self.name
 
+    def validate(self):
+        if not self.validator:
+            return
+        module_name, validator_func_name = str(self.validator).rsplit(".", 1)
+        module = importlib.import_module(module_name)
+        validator = getattr(module, validator_func_name)
+        if not validator(self.value):
+            raise ValueError("{} could not be validated by {} validator.".format(self.value, validator_func_name))
+
     def save(self, *args, **kwargs):
+        self.validate()
```

Comment:
I think that it would be better to do this in the model `clean` method instead of in the `save` method.

Reply:
@fabiocaccamo I've just pushed a commit with the requested changes. Thanks for the review!!

Reply:
Let me know if it's ok or it needs more improvements